varia/sass/base/_clearings.scss

```diff
@@ -21,7 +21,5 @@
 .site-header:after,
 .site-content:after,
 .site-footer:after {
-	content: "";
-	display: table;
```

Comment:
Haven’t tested this with Chrome on Android 10 yet but I think we may need to keep these 2 `content` and `display` rules. See here: https://css-tricks.com/snippets/css/clear-fix/